pkg/devfile/parser/data/v2/commands.go

```diff
@@ -57,3 +53,25 @@ func (d *DevfileV2) UpdateCommand(command v1.Command) {
 		}
 	}
 }
+
+// DeleteCommand removes the specified command
+func (d *DevfileV2) DeleteCommand(id string) error {
+
+	found := false
+	for i := range d.Commands {
+		if d.Commands[i].Id == id {
+			found = true
+			d.Commands = append(d.Commands[:i], d.Commands[i+1:]...)
+			break
```

Comment:
actually can just `return nil` at this point. and the `found` is not needed
same for the other delete functions